augly/video/augmenters/ffmpeg/hflip.py

```diff
@@ -1,22 +1,37 @@
 #!/usr/bin/env python3
 # Copyright (c) Facebook, Inc. and its affiliates.
 
-from typing import Tuple, Dict
+from typing import List, Optional
 
-from augly.video.augmenters.ffmpeg.base_augmenter import BaseFFMPEGAugmenter
-from ffmpeg.nodes import FilterableStream
+from augly.video.augmenters.ffmpeg.base_augmenter import BaseVidgearFFMPEGAugmenter
 
 
-class VideoAugmenterByHFlip(BaseFFMPEGAugmenter):
+class VideoAugmenterByHFlip(BaseVidgearFFMPEGAugmenter):
     def add_augmenter(
```

Comment:
Why don't you actually define `add_augmenter` in the base class, put all the code from `apply_ffmpeg_vidaug_augmenter` there, and then have a method that you override here (& in all vidgear ffmpeg augmenter classes) called `get_command(self, video_path: str, output_path: Optional[str] = None, **kwargs) -> List[str]` which just constructs the command? I think that makes more sense in terms of class structure.